date_range/readme/INSTALL.rst

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
I would remove this file as there are a lot of install methods and this doesn't make sense on apps.odoo.com. This section is for specifying pre-requirements or possible computation times on install.

Reply:
I agree, we must prepare a generic page somewhere explaining installation methods, there is no need to repeat what should be common knowledge is each addon.

Reply:
@pedrobaeza I propose the Odoo and the Pythonic way to install this addon.... For sure there are other ways to install this addon but I d'ont see why I should remove this section.

Reply:
I prefer the link to the possible installation methods, as said by @sbidoul, and it can be added by the generator. We already have https://odoo-community.org/blog/our-blog-1/post/how-to-install-oca-modules-79

Reply:
OK makes sense. 
Done